internal/commands/list_trusted_builders_test.go

```diff
@@ -19,7 +19,7 @@ import (
 	h "github.com/buildpacks/pack/testhelpers"
 )
 
-func TestListTrustedBuilders(t *testing.T) {
+func TestListTrustedBuildersCommand(t *testing.T) {
```

Comment:
Where is this change coming from across the board? We're already in the `commands` package, is it not reasonable to assume that all of our `Test*` cases are testing commands?

Reply:
It definitely is reasonable, but we have both patterns throughout the `commands` package. In this case, I find it useful to have the tests be `TextXCommand`, because I can then test the specific package by running `go test -run Command`, which I had been doing occasionally, and which I found didn't work as well as could be, given that a number of the commands didn't have that suffix.